src/main/java/com/hermesworld/ais/galapagos/changes/impl/ChangeBase.java

```diff
@@ -104,6 +104,14 @@ public static ChangeBase publishTopicSchemaVersion(String topicName, SchemaMetad
         return new PublishTopicSchemaVersionChange(topicName, schemaVersion);
     }
 
+    public static ChangeBase TopicProducerAddChange(String topicName, List<String> topicProducerIds) {
+        return new TopicProducerAddChange(topicName, topicProducerIds);
+    }
+
+    public static ChangeBase TopicProducerRemoveChange(String topicName, List<String> topicProducerRemoveId) {
```

Comment:
Why a list of producers here? According to rest of implementation, only one producer can be removed at a time.